docker-compose.yml

```diff
@@ -4,36 +4,78 @@ services:
     image: postgres:9.3
     container_name: api-java-postgres
 
-  possum:
-    image: registry.tld/possum:0.1.0-stable
-    container_name: api-java-possum
+  conjur:
+    image: cyberark/conjur:0.1.0-stable
+    container_name: api-java-conjur
     command: server -a cucumber
     environment:
+      PORT: 3000
       DATABASE_URL: postgres://postgres@postgres/postgres
       CONJUR_DATA_KEY: "W0BuL8iTr/7QvtjIluJbrb5LDAnmXzmcpxkqihO3dXA="
+    ports:
+      - "3000:3000"
+    expose:
+      - "3000"
     depends_on:
       - postgres
 
+  conjur-proxy-nginx:
+    build:
+      context: .
+      dockerfile: Dockerfile.nginx
+    entrypoint: nginx-debug -g 'daemon off;'
+    container_name: api-java-proxy
+    environment:
+        TERM: xterm
+    ports:
+    - "8080:80"
+    - "8443:443"
+    expose:
+      - "8080"
+      - "8443"
+    depends_on:
+      - conjur
+      - client
+
+
   client:
     image: conjurinc/cli5
-    container_name: api-java-possum-client
+    container_name: api-java-conjur-client
+    volumes:
+      - ./test-cert:/test-cert
     entrypoint: sleep
     command: infinity
     environment:
-      CONJUR_APPLIANCE_URL: http://possum
+      CONJUR_APPLIANCE_URL: http://conjur:3000
       DATABASE_URL: postgres://postgres@postgres/postgres
       CONJUR_ACCOUNT: cucumber
       CONJUR_AUTHN_LOGIN: admin
+    depends_on:
+      - conjur
 
   test:
     build:
       context: .
       dockerfile: Dockerfile.test
     container_name: api-java-test
+    image: api-java-tests
```

Comment:
If we don't plan on keeping the image, we shouldn't explicitly name it. I don't think we're pushing this anywhere, it's an ephemeral image. So we don't need `image`.

Reply:
I will change it